.github/workflows/continuous-integration-workflow.yml

```diff
@@ -69,9 +69,9 @@ jobs:
         uses: actions/checkout@v3
       - uses: actions/cache@v3
         with:
-          path: ~/.ccache
-          key: kokkos-${{ matrix.distro }}-${{ matrix.cxx }}-${{ matrix.cmake_build_type }}-${{ matrix.openmp }}-${github.ref}-${{ github.sha }}
-          restore-keys: kokkos-${{ matrix.distro }}-${{ matrix.cxx }}-${{ matrix.cmake_build_type }}-${{ matrix.openmp }}-${{github.ref}}
+          path: ~/.cache/ccache
```

Comment:
Do you know why the path is not the usual `${HOME}/.ccache`?
How is it set to something else?  I can't find a `CCACHE_DIR` environment variable.

Reply:
https://ccache.dev/manual/latest.html#_location_of_the_configuration_file

Reply:
Ubuntu image uses `ccache` version `4.5.1`. From https://ccache.dev/manual/4.5.html#_configuration_options:

> **cache_dir (CCACHE_DIR)**
> 
> This option specifies where ccache will keep its cached compiler outputs. The default is $XDG_CACHE_HOME/ccache if XDG_CACHE_HOME is set, **otherwise $HOME/.cache/ccache**. Exception: If the legacy directory $HOME/.ccache exists then that directory is the default.
> 

We do create the "legacy directory" in Ubuntu Dockerfile:
```Dockerfile
WORKDIR /home/kokkos
RUN mkdir .ccache
```
but when I ran `ccache` in verbose mode, on GitHub we have:
```
Summary:
  Cache directory:          /github/home/.cache/ccache
  Primary config:           /github/home/.config/ccache/ccache.conf
  Secondary config:         /etc/ccache.conf
```

Seems like because `/home/kokkos` is not actually `$HOME`, we fall into `$HOME/.cache/ccache` category.

Reply:
I think we should either set `XDG_CACHE_HOME` in the Dockerfile then and use it here or don't create `.ccache` in the Dockerfile.

Reply:
> I think we should either set `XDG_CACHE_HOME` in the Dockerfile then and use it here or don't create `.ccache` in the Dockerfile.

I agree, I'd opt for the latter to keep things simple. 
https://github.com/kokkos/ci-containers/pull/34 posted.